src/wib/WIBFrameProcessor.hpp

```diff
@@ -574,6 +627,15 @@ class WIBFrameProcessor : public TaskRawDataProcessorModel<types::WIB_SUPERCHUNK
   uint32_t m_offline_channel_base;           // NOLINT(build/unsigned)
   uint32_t m_offline_channel_base_induction; // NOLINT(build/unsigned)
 
+  // Frame error check
+  std::unique_ptr<folly::AtomicHashMap<uint32_t, // NOLINT(build/unsigned)
+  std::unique_ptr<dataformats::WIBFrame[]>>> m_err_frame_map;
```

Comment:
I don't want to see arrays like this in the readout code. The only exception where this is allowed is the overlay bitfields.